common/bitcointree/forfeit.go

```diff
@@ -15,9 +14,14 @@ func BuildForfeitTxs(
 	connectorAmount,
 	feeAmount uint64,
 	vtxoScript []byte,
-	aspPubKey *secp256k1.PublicKey,
+	aspAddress string,
```

Comment:
It seems inconsistent to work directly with Addresses at this level: why not using script directly? AFAIK network is not extracted here, and is constant for the bitcointree instance